### PR TITLE
Deployments page throws JS error, still referencing /images when should be getting imageStreams

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt or @csrwng review please

**Comment by jwforres** [comment]
its basically identical to what we do on the pods page except we have to pass in the pod templates from the deployments instead of actual pods

File: assets/app/scripts/controllers/deployments.js [hunk 1]
```diff
@@ -8,34 +8,44 @@
...
+        $scope.podTemplates[deploymentId] = deployment.spec.template;
+      });
+    };
+
     watches.push(DataService.watch("replicationcontrollers", $scope, function(deployments) {
       $scope.unfilteredDeployments = deployments.by("metadata.name");
       LabelFilter.addLabelSuggestionsFromResources($scope.unfilteredDeployments, $scope.labelSuggestions);
       LabelFilter.setLabelSuggestions($scope.labelSuggestions);
       $scope.deployments = LabelFilter.getLabelSelector().select($scope.unfilteredDeployments);
+      podTemplates();
```
Comment (csrwng) [inline assets/app/scripts/controllers/deployments.js]: Just a nit... it's not very clear what podTemplates() is doing from the name... maybe something like:
```
$scope.podTemplates = generatePodTemplates($scope.deployments)
```
would be more readable

**Comment by csrwng** [comment]
LGTM, minor nit above.

**Comment by jwforres** [comment]
[merge]

**Pull request merged**